tasks/main.yml

```diff
@@ -34,6 +34,14 @@
   with_items: project_pre_build_commands
   environment: project_environment
 
+- name: Check if vendor dir exists
```

Comment:
Checking if a previous install exists isn't needed when `project_copy_previous_composer_vendors` is off, so maybe add a `when` to this task?